apiary-metastore-events/apiary-hive-events/pom.xml

```diff
@@ -12,26 +12,19 @@
     <dependencies>
         <dependency>
             <groupId>org.apache.hive</groupId>
-            <artifactId>hive-metastore</artifactId>
-            <version>${hive.version}</version>
-            <scope>provided</scope>
```

Comment:
I have a feeling there might have been a reason we kept the scope at this level rather than in the parent's dependency management section but I'm not sure. Let's keep it like this for now as we don't need to redeclare the scope, we can always put it back if we find something doesn't work.